masterlist.yaml

```diff
@@ -5460,3 +5487,42 @@ plugins:
       - Delev
       - Invent
       - Relev
+  - name: 'BandolierForNPC - Cloaks of Skyrim Patch.esp'
+    req:
+      - 'Dr_Bandolier.esp'
+      - 'Cloaks.esp'
+    after:
+      - 'Dr_Bandolier.esp'
+      - 'Cloaks.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093'
+        name: Bandolier - Bags and Pouches for NPC - Cloaks of Skyrim Patch
+    tag:
+      - Invent
+  - name: 'BandolierForNPC - Winter is Coming Patch.esp'
+    req:
+      - 'Dr_Bandolier.esp'
+      - '1nivWICCloaks.esp'
+    after:
+      - 'Dr_Bandolier.esp'
+      - '1nivWICCloaks.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093'
+        name: Bandolier - Bags and Pouches for NPC - Winter is Coming Patch
+    tag:
+      - Invent
+  - name: 'BandolierForNPC - Winter is Coming and Cloaks of Skyrim Patch.esp'
+    req:
+      - 'Dr_Bandolier.esp'
+      - '1nivWICCloaks.esp'
+      - 'Cloaks.esp'
+    after:
+      - 'Dr_Bandolier.esp'
+      - '1nivWICCloaks.esp'
+      - 'Cloaks.esp'
+      - '1nivWICSkyCloaksPatch.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093'
+        name: Bandolier - Bags and Pouches for NPC - Winter is Coming and Cloaks of Skyrim Patch
```

Comment:
It's purely aesthetic but I usually keep `url` keys just under the `name` key for consistency.